subsys/bluetooth/host/conn.c

```diff
@@ -898,6 +898,22 @@ __maybe_unused static bool dont_have_methods(struct bt_conn *conn)
 		(conn->has_data == NULL);
 }
 
+static void move_head_to_tail(void)
+{
+	/* Move reference to the end to allow for another connection to be
+	 * scheduled the next time the TX processor is called.
+	 */
+	if (sys_slist_len(&bt_dev.le.conn_ready) == 1) {
```

Comment:
```suggestion
	if (sys_slist_peek_head(&bt_dev.le.conn_ready) == sys_slist_peek_tail(&bt_dev.le.conn_ready)) {
```

This matches the comment a bit better, and also only requires comparing two values (O(1)) instead of looping over the list (O(n))